resource-timing/crossorigin-sandwich-TAO.sub.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>This test validates resource timing information for a same-origin=>cross-origin=>same-origin redirect chain with Timing-Allow-Origin.</title>
+<link rel="author" title="Intel" href="http://www.intel.com/" />
```

Comment:
Not the right author

Reply:
As this is a copy of the existing tests, not sure what "author" should say, tbh. Seems best remove it entirely.